src/bls/bls.h

```diff
@@ -127,12 +135,9 @@ class CBLSWrapper
         return impl.Serialize(specificLegacyScheme);
     }
 
-    std::array<uint8_t, SerSize> ToBytes(const bool specificLegacyScheme) const
+    std::array<uint8_t, SerSize> ToBytes() const
     {
-        if (!fValid) {
-            return std::array<uint8_t, SerSize>{};
-        }
-        return impl.SerializeToArray(specificLegacyScheme);
+        return ToBytes(bls::bls_legacy_scheme.load());
```

Comment:
is it intentionally replaced `specificLegacyScheme` to global flag `bls::bls_legacy_scheme.load()` or that's just incorrectly resolved conflict?